blocked-edges/4.14.33-SRIOVFailedToConfigureVF.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.33
+from: ^4[.](13[.].*|14[.]([0-9]|1[0-9]|2[0-9]|3[0-2]))[+].*$
+url: https://issues.redhat.com/browse/NHE-1171
+name: SRIOVFailedToConfigureVF
+message: |-
+  Pods using a secondary interface of SRI-OV VF will stop coming up.
```

Comment:
I am not sure of the exact best wording at the moment as an impact statement is not proposed, and I do not understand the issue too much.

Reply:
The message was updated to `On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF will fail to create a pod sandbox and thus will not function.`.